packages/theme/composables/useCurrency/index.ts

```diff
@@ -11,14 +11,14 @@ const useCurrency = (): UseCurrency => {
   const configStore = useConfigStore();
   const currency = computed(() => configStore.currency);
 
-  const load = async (params?: ComposableFunctionArgs<CustomQuery>) => {
+  const load = async (params?: ComposableFunctionArgs<{ currency: string }>) => {
```

Comment:
The type received by `ComposableFunctionArgs` is the custom query, but something to combine with the object that has custom query.

`ComposableFunctionArgs<{ currency: string }>` became `{ customQuery?: CustomQuery, currency: string }` instead of `{ customQuery?: { currency: string } }`, and since we're not using `currency` in the method I think we can just pass an empty object instead.

```suggestion
  const load = async (params?: ComposableFunctionArgs<{}>) => {
```